src/models/entities/Context.ts

```diff
@@ -42,5 +42,10 @@ export class Context extends BaseEntity implements IContext {
     )
     references?: Reference[];
 
+    // Constructor
+    constructor() {
+      super();
```

Comment:
ins't this implicit?

Reply:
Yes, had put it in as wanted to initialise the class properly with an empty array - moved that to the initialise() method. It can be removed.